cascadia.yaml

```diff
@@ -46,6 +52,21 @@ Options:
     Flag: b,base
     Usage: base href tag used in the wrapped up html
 
+  - Name: RawAttr
+    Type: string
+    Flag: 'a,attr,raw-attr'
```

Comment:
I would remove that `attr`, because I think it would confuse people than helpful. `raw-attr` (that goes with `raw-text`) should be good enough. But, your call.